From 95e837d9595d83938fd1f1d3a34a3738ccf9968c Mon Sep 17 00:00:00 2001
From: "jonghen.han" <jonghen.han@lge.com>
Date: Thu, 25 Feb 2021 15:58:09 +0900
Subject: [PATCH] webOS: Fix bug that API is not called

:Release Notes:
Fix bug that API is not called

:Detailed Notes:
This commit support to new ACG policy.

:Testing Performed:
- Build is successfully done.
- Verified that local build version.

:Issues Addressed:
---
 .../samples/webos/com.example.app.iotivity/appinfo.json     | 2 +-
 .../com.example.service.iotivity.client.manifest.json       | 6 ++++++
 .../com.example.service.iotivity.server.manifest.json       | 6 ++++++
 3 files changed, 13 insertions(+), 1 deletion(-)

diff --git a/resource/csdk/stack/samples/webos/com.example.app.iotivity/appinfo.json b/resource/csdk/stack/samples/webos/com.example.app.iotivity/appinfo.json
index 9f46b8fe1..c8578c34a 100644
--- a/resource/csdk/stack/samples/webos/com.example.app.iotivity/appinfo.json
+++ b/resource/csdk/stack/samples/webos/com.example.app.iotivity/appinfo.json
@@ -7,5 +7,5 @@
     "title": "IoTivity Sampler",
     "icon": "icon.png",
     "uiRevision": 2,
-    "requiredPermissions":["com.example.service.iotivity.client.group", "com.example.service.iotivity.server.group"]
+    "requiredPermissions":["com.example.service.iotivity.client.group", "com.example.service.iotivity.server.group","notification.operation"]
 }
diff --git a/resource/csdk/stack/samples/webos/com.example.app.iotivity/services/com.example.service.iotivity.client/files/sysbus/com.example.service.iotivity.client.manifest.json b/resource/csdk/stack/samples/webos/com.example.app.iotivity/services/com.example.service.iotivity.client/files/sysbus/com.example.service.iotivity.client.manifest.json
index e62142f6c..dc3005588 100644
--- a/resource/csdk/stack/samples/webos/com.example.app.iotivity/services/com.example.service.iotivity.client/files/sysbus/com.example.service.iotivity.client.manifest.json
+++ b/resource/csdk/stack/samples/webos/com.example.app.iotivity/services/com.example.service.iotivity.client/files/sysbus/com.example.service.iotivity.client.manifest.json
@@ -9,5 +9,11 @@
     ],
     "serviceFiles": [
         "/usr/share/luna-service2/services.d/com.example.service.iotivity.client.service"
+    ],
+    "apiPermissionFiles": [
+        "/usr/share/luna-service2/api-permissions.d/com.example.service.iotivity.client.api.json"
+    ],
+    "groupsFiles": [
+        "/usr/share/luna-service2/groups.d/com.example.service.iotivity.client.groups.json"
     ]
 }
\ No newline at end of file
diff --git a/resource/csdk/stack/samples/webos/com.example.app.iotivity/services/com.example.service.iotivity.server/files/sysbus/com.example.service.iotivity.server.manifest.json b/resource/csdk/stack/samples/webos/com.example.app.iotivity/services/com.example.service.iotivity.server/files/sysbus/com.example.service.iotivity.server.manifest.json
index 0234aefe9..f2a7a36dc 100644
--- a/resource/csdk/stack/samples/webos/com.example.app.iotivity/services/com.example.service.iotivity.server/files/sysbus/com.example.service.iotivity.server.manifest.json
+++ b/resource/csdk/stack/samples/webos/com.example.app.iotivity/services/com.example.service.iotivity.server/files/sysbus/com.example.service.iotivity.server.manifest.json
@@ -9,5 +9,11 @@
     ],
     "serviceFiles": [
         "/usr/share/luna-service2/services.d/com.example.service.iotivity.server.service"
+    ],
+    "apiPermissionFiles": [
+        "/usr/share/luna-service2/api-permissions.d/com.example.service.iotivity.server.api.json"
+    ],
+    "groupsFiles": [
+        "/usr/share/luna-service2/groups.d/com.example.service.iotivity.server.groups.json"
     ]
 }
\ No newline at end of file
